Cargo.lock

```diff
@@ -1583,7 +1597,7 @@ dependencies = [
  "ethcore-sync",
  "ethereum-types",
  "kvdb",
- "kvdb-rocksdb",
+ "kvdb-rocksdb 0.6.0",
```

Comment:
Upgraded `kvdb-rocksdb` to 0.6 to improve cache stats reporting.
unfortunately `kvdb-rocksdb 0.5.0` is still there because of secret-store

Reply:
Ok, there is not possible to bump it in secret-store too?

Reply:
There are some breaking changes https://github.com/paritytech/secret-store#secret-store-20

Reply:
I guess it shouldn't build two versions of rocksdb if secretstore feature is disabled (which is the default)

Reply:
Yeah, I think that is correct